analytics-api-core/src/main/scala/org/ekstep/analytics/api/service/JobAPIService.scala

```diff
@@ -199,8 +193,10 @@ object JobAPIService {
   private def _validateRequest(channel: String, eventType: String, from: String, to: String)(implicit config: Config): Map[String, String] = {
 
     APILogger.log("Validating Request", Option(Map("channel" -> channel, "eventType" -> eventType, "from" -> from, "to" -> to)))
-    if (!EVENT_TYPES.contains(eventType)) {
-      return Map("status" -> "false", "message" -> "Please provide 'eventType' value should be one of these -> ('raw' or 'summary' or 'metrics', or 'failed') in your request URL")
+    val datasetTypes = config.getObject("channel.data_exhaust.dataset").unwrapped().keySet()
+    datasetTypes.remove("default")
+    if (!datasetTypes.contains(eventType)) {
+      return Map("status" -> "false", "message" -> "Please provide valid datasetId in request URL")
```

Comment:
Do we need this check? the result will anyway return a blank list right?

Reply:
We need this check as we are getting bucket and basePath based on dataset id from config json -
`val bucket = if (config.getString(s"channel.data_exhaust.dataset.$datasetId.bucket").isEmpty) config.getString("channel.data_exhaust.dataset.default.bucket") else config.getString(s"channel.data_exhaust.dataset.$datasetId.bucket")`
If that check is removed it will fail for fetching these configs. Fixing this will return files for default config is that fine?

Reply:
Removing this check will provide files for default config. Will that be fine?

Reply:
Removed check on datasetId